udf-compiler/src/main/scala/com/nvidia/spark/udf/Plugin.scala

```diff
@@ -17,8 +17,8 @@
 package com.nvidia.spark.udf
 
 import ai.rapids.cudf.{NvtxColor, NvtxRange}
+import com.nvidia.spark.RapidsUDF
 import com.nvidia.spark.rapids.RapidsConf
-
```

Comment:
This empty line is required, otherwise `mvn verify` would fail.

Reply:
Done.